app/controllers/passengers_controller.rb

```diff
@@ -16,6 +16,11 @@ def toggle_archive
     redirect_to passengers_url, notice: 'Passenger successfully updated'
   end
 
+  def check_existing
+    @passenger = User.find_by(spire: params[:spire_id])
+    render partial: 'check_existing'
```

Comment:
Doesn't this controller action _always_ render this partial?  Maybe I'm missing something?

Reply:
yes and it finds a user not a passenger